tests/e2e-leg-6/verify-vrpq-conditions/04-create-archive-and-restorepoints.yaml

```diff
@@ -0,0 +1,50 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: kubectl exec -n $NAMESPACE archive-query-sc1-0 -- vsql -w superuser -tAc "CREATE TABLE public.test_table (val INTEGER);"
```

Comment:
These would be better suited if everything went into a configMap/pod and run there. See tests/e2e-leg-3/client-access/35-verify-memory-limit.yaml.